index.js

```diff
@@ -12,3 +12,14 @@ I need this code, just don't know where, perhaps should make some middleware, do
 
 Go code!
 */
+
+const envSetup = require('dotenv');
+envSetup.config();
+
+const server = require('./server.js');
+
+const port = process.env.PORT || 5000;
+
+server.listen(port, () => {
+    console.log(`\n***server listening on port ${port}***`);
+})
```

Comment:
great setup here, you're ready to deploy with no further modifications for this file